File: src/main/java/com/nunchuk/android/type/MiniscriptTimelockBasedHelper.java
package com.nunchuk.android.type;

public final class MiniscriptTimelockBasedHelper {
    private MiniscriptTimelockBasedHelper() {}

    public static MiniscriptTimelockBased from(final int ordinal) {
        return CollectionUtil.fromOrdinal(MiniscriptTimelockBased.values(), ordinal, MiniscriptTimelockBased.NONE);
    }
} 

File: src/main/java/com/nunchuk/android/type/MiniscriptTimelockBasedHelper.kt
package com.nunchuk.android.type

object MiniscriptTimelockBasedHelper {
    @JvmStatic
    fun from(ordinal: Int): MiniscriptTimelockBased =
        CollectionUtil.fromOrdinal(MiniscriptTimelockBased.entries, ordinal, MiniscriptTimelockBased.NONE)
}
